data/dataspace.json

```diff
@@ -11,7 +11,111 @@
     "members": 1178,
     "datasets": 822,
     "services": 674,
-    "governanceUrl": "https://xd.adobe.com/view/31a3d2a5-9f07-4e31-a612-20059ff929a5-64f0/screen/d8a45cb1-7433-40b3-b6ce-4237bfcd0678/?fullscreen",
+    "governance": [
```

Comment:
Why governance is an array ? There's a one-to-one relationship between a dataspace and a governance, so I'd expect to see a single object here.

Reply:
Since the governance is an abstraction of the smart contract I was trying to have the governance be as malleable as possible. With that said after discussing with @amimart I agree it should rather be an object. I will implement this

Reply:
Perfect. Don't go too far in the abstraction because the way we will represent the governance will be completely different (we will set up a workshop to discuss this). Focus on the information required by the ui (backend for frontend approach) trying to keep the general concepts. 👍

Reply:
Yes that's a good point. For the moment I have turned `governance` into an object as requested with the following keys:
- `accessControl: GovernanceContent[]`
- `dataManagement: GovernanceContent[]`
- `serviceManagement: GovernanceContent[]`
- `businessModel: GovernanceContent[]`
- `governance : GovernanceContent[]`

I also took the opportunity to internationalize the governance categories.